src/main/java/com/fewlaps/quitnowcache/QNCache.java

```diff
@@ -80,6 +83,33 @@ public void set(String key, T value, long keepAliveInMillis) {
         }
     }
 
+    public List<String> listCachedKeysStartingWith(String keyStartingWith) {
+       	List<String> keys = new ArrayList<String>();
+    	keyStartingWith = getEffectiveKey(keyStartingWith);    	
```

Comment:
Why not calling it "key" or "effectiveKey" ? Reusing an input parameter is not an issue in this case, but if we call it "effectiveKey" we will be sure that the key has been translated when using it.